README.md

```diff
@@ -23,7 +23,34 @@ A WIP Rust implementation of [Messaging Layer Security](https://github.com/mlswg
 
 ### Dependencies
 
-OpenMLS relies on [EverCrypt](https://github.com/project-everest/hacl-star/tree/master/providers/evercrypt), a high-performance, cross-platform, formally verified modern cryptographic provider through [EverCrypt Rust bindings](https://crates.io/crates/evercrypt).
+#### Cryptography
+
+OpenMLS does not implement its own cryptographic primitives.
+Instead, it relies on existing implementations of the cryptographic primitives used.
+There are two different cryptography backends supported right now.
+
+##### Rust Crypto
+
+This is the default cryptographic backend, using [hkdf], [sha2], [p256], [p384], [x25519-dalek-ng], [ed25519-dalek] [chacha20poly1305], [aes-gcm].
+
+
+##### Evercrypt
+
+[EverCrypt] provides a formally verified cryptographic backend through the through [EverCrypt Rust bindings] to OpenMLS.
+The default features have to disabled and the `evercrypt-backend` feature has to be enabled.
```

Comment:
"have to be"/ "must be"

Reply:
"have to be" would be wrong. But happy to do "must"

```suggestion
The default features have to disabled and the `evercrypt-backend` feature must be enabled.
```